py/circuitpy_mpconfig.mk

```diff
@@ -58,6 +58,10 @@ CFLAGS += -DCIRCUITPY=$(CIRCUITPY)
 CIRCUITPY_FULL_BUILD ?= 1
 CFLAGS += -DCIRCUITPY_FULL_BUILD=$(CIRCUITPY_FULL_BUILD)
```

Comment:
Why not use CIRCUITPY_FULL_BUILD instead? This is essentially CIRCUITPY_NOTFLASH_LIMITED already.

Reply:
I thought about it, but I thought the last time I looked at FULL_BUILD, it had a lot of side effects. I was worried that turning FULL_BUILD off would turn off other things that wouldn't be noticed until someone tried to upgrade their board.

I did notice some of the other M0 boards were using the FULL_BUILD setting so if you think that's a better approach, I'm happy to get this working using that parameter.

Reply:
As I think about it more, I think I'm realizing that using the new parameter going forward as I had envisioned it would also pick up multiple side effects just like the FULL_BUILD parameter. I think you're right and they're redundant. I'll go ahead and make the change. Thanks!

Reply:
All set, this is using the CIRCUITPY_FULL_BUILD parameter now.